compiler/vmops.nim

```diff
@@ -148,3 +148,8 @@ proc registerAdditionalOps*(c: PCtx) =
     let n = getNode(a, 0)
     if n.kind != nkSym: raise newException(ValueError, "node is not a symbol")
     setResult(a, $symBodyDigest(c.graph, n.sym))
+
+  registerCallback c, "stdlib.macros.isExported", proc(a: VmArgs) {.nimcall.} =
+    let n = getNode(a, 0)
+    if n.kind != nkSym: raise newException(ValueError, "node is not a symbol")
```

Comment:
The VM doesn't handle exceptions... Simply return ``false`` then?

Reply:
I would use ``template stackTrace(c: PCtx, tos: PStackFrame, pc: int, msg: string)`` here.

Reply:
Or that, yes.

Reply:
Done.